src/data/markdown/docs/30 xk6-browser/01 xk6-browser/02 BrowserContext/newPage.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'newPage()'
+excerpt: 'Creates a new page inside this BrowserContext.'
+---
+
+Uses the `BrowserContext` to create a new [Page](/javascript-api/xk6-browser/page/) and returns it.
```

Comment:
We need to add a code example here too

Reply:
Resolved in: 938e08855aaf6e2b256cd4a7e192ea9a6aa5f5d5